### ISSUE TITLE
Updating in Linux and Windows

### ISSUE THREAD

**Issue by cristianfraser** [edjafarov/node-webkit-updater#67]
I'm not sure if I should open an issue for this, but this seems the best place to ask this. 
What is "Note that compressed apps are assumed to be downloaded in the format produced by node-webkit-builder" supposed to mean?

In OS X, it's fairly simply to use this as the file to be replaced is one. But for the Windows and Linux buils, node-webkit-updater outputs 3-5 more files besides the executable. Do this mean the .zip to be downloaded has to include those extra files as well? If that's the case, is it necessary that the binaries are in the same folder as the other files?

tl;dr What am I supposed to do with the non-executable files in the Linux and Windows output of `node-webkit-builder`?

**Comment by adam-lynch** [comment]
Sorry for not seeing this... yes, include everything in the same directory.
